packages/mui-icons-material/package.json

```diff
@@ -46,7 +46,9 @@
     "typescript": "tslint -p tsconfig.json \"src/**/*.{ts,tsx}\""
   },
   "peerDependencies": {
-    "@mui/material": "^5.0.0",
+    "@mui/system": "^5.11.2",
```

Comment:
Is there a singleton on this one? I would imagine that the system can be a direct dependency

Also, per https://github.com/mui/material-ui/pull/32623 we should add a peer dependent on emotion.

Reply:
Yes, it's my bad, I will update the dependencies.